tests/unit/module_utils/test_role_utils.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible_collections.sensu.sensu_go.plugins.module_utils import role_utils
+
+
+class TestSortRules:
+    def test_sort_rules(self):
+        assert role_utils._sort_rules(
+            [{'a': [2, 1]}, {'b': [4, 3]}]
+        ) == [{'a': [1, 2]}, {'b': [3, 4]}]
+
+    def test_sort_none_value(self):
+        assert role_utils._sort_rules(
+            [{'a': [2, 1]}, {'b': None}]
+        ) == [{'a': [1, 2]}, {'b': None}]
+
+
+class TestDoRulesDiffer:
+    def test_empty_values(self):
+        assert role_utils._do_rules_differ(
+            [{'a': []}],
+            [{'a': []}]
+        ) is False
+
+    def test_rules_when_value_is_none(self):
+        assert role_utils._do_rules_differ(
+            [{'a': [2, 1]}, {'c': None}],
+            [{'a': [1, 3]}, {'b': None}]
+        ) is True
+
+    def test_rules_are_different(self):
+        assert role_utils._do_rules_differ(
+            [{'a': [2, 1]}],
+            [{'a': [1, 3]}]
+        ) is True
+
+    def test_rules_more_rules_in_current(self):
+        assert role_utils._do_rules_differ(
+            [{'a': [2, 1]}, {'b': [3, 4]}],
+            [{'a': [1, 2]}]
+        ) is True
+
+    def test_rules_with_additional_keys_in_current(self):
```

Comment:
I may be wrong, but is this not basically the same test as `test_rules_more_rules_in_current`?

Reply:
heh, you are right :sweat_smile: